commands/bridge_token_add.go

```diff
@@ -0,0 +1,39 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/MichaelMure/git-bug/bridge/core"
+)
+
+var (
+	bridgeToken core.Token
+)
+
+func runBridgeTokenAdd(cmd *cobra.Command, args []string) error {
+	_ = bridgeToken.Id() // TODO: a better design to avoid doing this
```

Comment:
it's better to have individual variables for the parameters and to use the constructor